packages/chain-adapters/src/utils/index.ts

```diff
@@ -66,3 +67,5 @@ export const chainIdToChainLabel = (chainId: ChainId): string => {
       throw new Error(`chainNamespace ${chainNamespace} not supported.`)
   }
 }
+
+export const convertNumberToHex = (value: string): string => BigNumber.from(value).toHexString()
```

Comment:
The magic of this PR - our previous `utils.hexlify(utils.hexlify(bnOrZero(value).toNumber()))` logic was an absolute derpy one - if JS numbers could be big enough that `.toNumber()` works on our base units, we wouldn't need a big numbers lib to start with